src/index.ts

```diff
@@ -489,11 +489,12 @@ export const open = (options: {
   };
 };
 
-export const moveAssetsDatabase = (
-  dbName: string,
-  extension: string
-): boolean => {
-  return NativeModules.OPSQLite.moveAssetsDatabase(dbName, extension);
+export const moveAssetsDatabase = (args: {
```

Comment:
This method's native implementation is now async so the definition here needs to be updated to reflect that.

Reply:
ah yeah, missed that. Thanks!